content/id/docs/concepts/configuration/secret.md

```diff
@@ -0,0 +1,1062 @@
+---
+title: Secret
+content_template: templates/concept
+feature:
+  title: Secret dan manajemen konfigurasi
+  description: >
+    Menerapkan serta mengubah secret serta konfigurasi aplikasi tanpa melakukan perubahan pada image kamu serta mencegah tereksposnya secret yang kamu miliki pada konfigurasi.
+weight: 50
+---
+
+
+{{% capture overview %}}
+
+Objek `secret` pada Kubernetes mengizinkan kamu menyimpan dan mengatur informasi yang sifatnya sensitif, seperti
+_password_, token OAuth, dan ssh _keys_. Menyimpan informasi yang sifatnya sensitif ini ke dalam `secret`
+cenderung lebih aman dan fleksible jika dibandingkan dengan menyimpan informasi tersebut secara apa adanya pada
+definisi {{< glossary_tooltip term_id="pod" >}} atau di dalam {{< glossary_tooltip text="container image" term_id="image" >}}.
+Silahkan lihat [Dokumen desain Secret](https://git.k8s.io/community/contributors/design-proposals/auth/secrets.md) untuk informasi yang sifatnya mendetail.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Ikhtisar Secret
+
+Sebuah Secret merupakan sebuah objek yang mengandung informasi yang sifatnya
+sensitif, seperti _password_, token, atau _key_. Informasi tersebut sebenarnya bisa saja
+disimpan di dalam spesifikasi Pod atau _image_; meskipun demikian, melakukan penyimpanan
+di dalam objek Secret mengizinkan pengguna untuk memiliki kontrol lebih lanjut mengenai
+bagaimana Secret ini disimpan, serta mencegah tereksposnya informasi sensitif secara
+tidak disengaja.
+
+Baik pengguna dan sistem memiliki kemampuan untuk membuat objek Secret.
+
+Untuk menggunakan Secret, sebuah Pod haruslah merujuk pada Secret tersebut.
+Sebuah Secret dapat digunakan di dalam sebuah Pod melalui dua cara:
+sebagai _file_ yang ada di dalam _volume_ {{< glossary_tooltip text="volume" term_id="volume" >}}
```

Comment:
same with this, glossary lines are breaking

![image](https://user-images.githubusercontent.com/3192347/61197631-43bfe980-a700-11e9-877f-5a0099950b21.png)